hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/scm/ReconStorageContainerManagerFacade.java

```diff
@@ -182,6 +190,21 @@ public ReconStorageContainerManagerFacade(OzoneConfiguration conf,
         .setSCM(this)
         .build();
     this.ozoneConfiguration = getReconScmConfiguration(conf);
+    long scmClientRPCTimeOut = ozoneConfiguration.getTimeDuration(
+        OZONE_RECON_SCM_CLIENT_RPC_TIME_OUT_KEY,
+        OZONE_RECON_SCM_CLIENT_RPC_TIME_OUT_DEFAULT,
+        TimeUnit.MILLISECONDS);
+    long scmClientMaxRetryTimeOut = ozoneConfiguration.getTimeDuration(
+        OZONE_RECON_SCM_CLIENT_MAX_RETRY_TIMEOUT_KEY,
+        OZONE_RECON_SCM_CLIENT_MAX_RETRY_TIMEOUT_DEFAULT,
+        TimeUnit.MILLISECONDS);
+    int scmClientFailOverMaxRetryCount = ozoneConfiguration.getInt(
+        OZONE_RECON_SCM_CLIENT_FAILOVER_MAX_RETRY_KEY,
+        OZONE_RECON_SCM_CLIENT_FAILOVER_MAX_RETRY_DEFAULT);
+    SCMClientConfig scmClientConfig = conf.getObject(SCMClientConfig.class);
```

Comment:
do this update for scmClientConfig is getting reflected? as getObject will return new instance. May need update hdds property value directly using recon property to overwrite.

Reply:
Thanks @sumitagrawl for review.  Done the changes.